rpc/evm.go

```diff
@@ -199,7 +199,7 @@ func (e *EvmExecutor) sendCall() (*core.ExecutionResult, error) {
 // sendEstimateGas executes estimateGas method in the EvmExecutor
 // It calculates how much gas would transaction need if it was executed
 func (e *EvmExecutor) sendEstimateGas() (hexutil.Uint64, error) {
-	panic("not implemented")
+	return hexutil.Uint64(0), fmt.Errorf("not implemented")
```

Comment:
This panic can be kept. `sendEstimateGas` should not be executed. This is the only valid kind of panic.